spring-integration-file/src/test/java/org/springframework/integration/file/config/FileOutboundChannelAdapterParserTests-context.xml

```diff
@@ -49,6 +49,20 @@
 								   auto-startup="false"
 								   directory="${java.io.tmpdir}"/>
 
+	<file:outbound-channel-adapter id="adapterWithFlushing"
+								   channel="testChannel"
+								   order="555"
+								   mode="APPEND_NO_FLUSH"
+								   flush-interval="12345"
+								   flush-predicate="predicate"
```

Comment:
I wonder if one day SF will support Lambda for such an options in the XML:
```
flush-predicate="(p, t, m) -> true"
```
:smile: